actors/util.go

```diff
@@ -65,6 +65,8 @@ const (
 )
 
 var (
+	// 1<<63-1 nanoseconds equals about 292 years, which isn't truly "forever" but is the longest valid time.Duration value in Golang.
```

Comment:
TIL

Reply:
Yeah I a making use of the longest time.Duration value.

Reply:
This should suffice most use cases.